ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -427,12 +424,12 @@ If server is connected to only one leaf to the fabric, following template can be
 
 ```yaml
 servers:
-  A-PR01-DMZ-POD01:     # Server name
-    rack: POD01         # Informational RACK
+  A-PR01-DMZ-POD01:                 # Server name
+    rack: POD01                     # Informational RACK
     adapters:
       - type: nic
-        server_ports: [Eth0]        # Server port to connect
-        switch_ports: [Ethernet3]   # Switch port to connect server
+        endpoint_ports: [Eth0]      # Port on server/endpoint
```

Comment:
In the hole block, we should converge to use either server, endpoint or both to avoid any confusion.

Reply:
I think we might deprecate ```server_ports```. So I used ```endpoint_ports``` in the example.

Reply:
I will update the comments.